pacai/student/searchAgents.py

```diff
@@ -201,7 +198,7 @@ def findPathToClosestDot(self, gameState):
         # problem = AnyFoodSearchProblem(gameState)
 
         # *** Your Code Here ***
-        util.raiseNotDefined()
+        raise NotImplementedError("findPathToClosestDot")
         return None
```

Comment:
Since we are raising now (instead of calling a method), we don't need to return.

Reply:
Ok, yeah I wasn't exactly sure if I should keep the return in there. I left it there so it's clear to the students that the method does in fact return a path